### ISSUE TITLE
Add fork() method to support multiprocess

### ISSUE THREAD

**Issue by vincentdchan** [veegee/guv#22]
This an issue of enhancement

I think `guv` should add a fork() method to support multiprocess.

**Comment by veegee** [comment]
I agree, added an enhancement tag to this issue. I'll keep it open until it's added.
